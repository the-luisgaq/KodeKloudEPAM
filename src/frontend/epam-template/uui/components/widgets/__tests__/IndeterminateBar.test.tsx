import React from 'react';
import { IndeterminateBar } from '../IndeterminateBar';
import { render } from '@epam/uui-test-utils';

describe('ProgressBar', () => {
    it('should be rendered correctly', () => {
        const { asFragment } = render(<IndeterminateBar />);
        expect(asFragment()).toMatchSnapshot();
    });

    it('should be rendered correctly with props', () => {
        const { asFragment } = render(<IndeterminateBar size="24" />);
        expect(asFragment()).toMatchSnapshot();
    });
});
