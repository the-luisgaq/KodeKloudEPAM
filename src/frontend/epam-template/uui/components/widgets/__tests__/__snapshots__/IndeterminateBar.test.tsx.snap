// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar should be rendered correctly 1`] = `
<DocumentFragment>
  <div
    class="uui-indeterminate_bar root size-12"
  >
    <div
      class="progressBar"
    />
  </div>
</DocumentFragment>
`;

exports[`ProgressBar should be rendered correctly with props 1`] = `
<DocumentFragment>
  <div
    class="uui-indeterminate_bar root size-24"
  >
    <div
      class="progressBar"
    />
  </div>
</DocumentFragment>
`;
